src/pe_reports/data/db_query.py

```diff
@@ -0,0 +1,193 @@
+"""Query the PE postgres database."""
+
+# Standard Python Libraries
+import logging
+import sys
+
+# Third-Party Libraries
+import pandas as pd
+import psycopg2
+from psycopg2 import OperationalError
+from psycopg2.extensions import AsIs
+
+from .config import config
+
+logging.basicConfig(format="%(asctime)-15s %(levelname)s %(message)s", level="INFO")
+
+CONN_PARAMS_DIC = config()
+
+
+def show_psycopg2_exception(err):
+    """Handle errors for postgres issues."""
+    err_type, traceback = sys.exc_info()
+    line_n = traceback.tb_lineno
+    logging.error(f"\npsycopg2 ERROR: {err} on line number: {line_n}")
+    logging.error(f"psycopg2 traceback: {traceback} -- type: {err_type}")
+    logging.error(f"\nextensions.Diagnostics: {err}")
+
+    logging.error(f"pgerror: {err}")
+
+    logging.error(f"pgcode: {err}\n")
+
+
+def connect():
+    """Connect to postgres database."""
+    conn = None
+    try:
+        logging.info("Connecting to the PostgreSQL......")
+
+        conn = psycopg2.connect(**CONN_PARAMS_DIC)
+        logging.info("Connection successful................\n")
+
+    except OperationalError as err:
+        show_psycopg2_exception(err)
+        conn = None
+    return conn
+
+
+def close(conn):
+    """Close connection to postgres."""
+    conn.close()
+    return
+
+
+def get_orgs(conn):
+    """Query orgs table."""
+    try:
+        cur = conn.cursor()
+        sql = """SELECT * FROM organizations"""
+        cur.execute(sql)
+        pe_orgs = cur.fetchall()
+        cur.close()
+        return pe_orgs
+    except (Exception, psycopg2.DatabaseError) as error:
+        logging.error(f"There was a problem with your database query {error}")
+    finally:
+        if conn is not None:
+            close(conn)
+
+
+def query_hibp_view(conn, org_uid, start_date, end_date):
+    """Query hibp table."""
```

Comment:
```suggestion
    """Query 'Have I Been Pwned?' table."""
```
Although it looks like the table name here is `vw_breach_complete` so maybe `Query 'Have I Been Pwned?' data.` makes more sense here?